src/ir_Mitsubishi.cpp

```diff
@@ -1186,3 +1198,449 @@ String IRMitsubishi136::toString(void) {
   result += addBoolToString(getQuiet(), F("Quiet"));
   return result;
 }
+
+
+#if SEND_MITSUBISHI112
+// Send a Mitsubishi112 A/C message.
+//
+// Args:
+//   data: An array of bytes containing the IR command.
+//   nbytes: Nr. of bytes of data in the array. (>=kMitsubishi112StateLength)
+//   repeat: Nr. of times the message is to be repeated.
+//          (Default = kMitsubishi112MinRepeat).
+//
+// Status: ALPHA / Probably working. Needs to be tested against a real device.
+//
+// Ref:
+//   https://github.com/crankyoldgit/IRremoteESP8266/issues/888
+void IRsend::sendMitsubishi112(const unsigned char data[],
+                               const uint16_t nbytes,
+                               const uint16_t repeat) {
+  if (nbytes < kMitsubishi112StateLength)
+    return;  // Not enough bytes to send a proper message.
+
+  sendGeneric(kMitsubishi112HdrMark, kMitsubishi112HdrSpace,
+              kMitsubishi112BitMark, kMitsubishi112OneSpace,
+              kMitsubishi112BitMark, kMitsubishi112ZeroSpace,
+              kMitsubishi112BitMark, kMitsubishi112Gap,
+              data, nbytes, 38, false, repeat, 50);
+}
+#endif  // SEND_MITSUBISHI112
+
+#if DECODE_MITSUBISHI112
+// Decode the supplied Mitsubishi112 message.
+//
+// Args:
+//   results: Ptr to the data to decode and where to store the decode result.
+//   nbits:   Nr. of data bits to expect.
+//   strict:  Flag indicating if we should perform strict matching.
+// Returns:
+//   boolean: True if it can decode it, false if it can't.
+//
+// Status: STABLE / Reported as working.
+//
+// Ref:
+// FIXME
```

Comment:
Feel free to reference this PR or your original issue number #945 .. or leave blank. I normally add this so I can find the canonical "source" of where I got the data to add the protocol. As you added it, you're the reference. So not really needed. ;-)